README.md

```diff
@@ -2,31 +2,630 @@
+`Person()` is called.
+
+![constructor breakpoint](https://curriculum-content.s3.amazonaws.com/6677/pillars/breakpoint_constructor.png)
+
+
+Press "Step Into" to execute the statement `Teacher teacher = new Teacher()`.
+
+The call stack shows a new frame named `<init>`, which represents the `Teacher()`
+constructor method.  We can see control is transferred to the first statement
+inside the `Teacher()` constructor, which is the call `super();`.
+
+![teacher constructor](https://curriculum-content.s3.amazonaws.com/6677/pillars/teacher_constructor.png)
+
+Press "Step Into" to execute the statement `super();`.
+
+The call stack shows another new frame named `<init>`, which represents the `Person()`
+constructor method.  We can see control is transferred to the first statement
+inside the `Person()` constructor, which again is the call `super();`.
+
+![person constructor](https://curriculum-content.s3.amazonaws.com/6677/pillars/person_constructor.png)
+
+Executing the call `super();` within the `Person` constructor calls the `Object`
+constructor, which we can't step into.  We can "Step Over" the `super();` statement.
+You can continue stepping over the print statement and return 
+to the `Teacher` constructor. Continue stepping back to the `main` method
+or press the "Resume Program" button to execute the rest of the code.
+
+## Calling superclass constructors with arguments
+
+The no-args constructors initialize the fields to their default values.
+Let's update `Person` to add a constructor with parameters so we can initialize
+the fields to specific values.  We'll add a print statement in the constructor
+as well just so we can see when it is called, versus the no-args constructor.
+Defining multiple constructors is an example of **method overloading**,
+as the class has two methods with the same name but different parameter lists.
+
+```java
+public class Person {
+
+    private String name;
+    private int age;
+
+    public Person() {
+        super();   //calls Object() constructor
+        System.out.println("Person() constructor");
+    }
+
+    public Person(String name, int age) {
+        super();   //calls Object() constructor
+        System.out.println("Person(String name, int age) constructor");
+        this.name = name;
+        this.age = age;
+    }
+
+    // other methods
+}
+```
+
+We'll update the `main` to call this constructor to initialize the `Person`
+object's state:
+
+```java
+public class Main {
+
+    public static void main(String[] args) {
+        //Person instance
+        System.out.println("Creating Person");
+        Person person = new Person("Kai", 32);
+        System.out.println("Person state:" + person); 
+
+        //Teacher instance, inherits fields and methods from Person
+        //...
+    }
+}
+```
+
+Running the code shows the `Person(String name, int age)` constructor
+executing for the call `new Person("Kai", 32);`.
```

Comment:
I don't think we need the word "for" in this sentence?

Reply:
fixed